Desktop/gui/jaspversionchecker.cpp

```diff
@@ -46,7 +46,7 @@ void JASPVersionChecker::downloadKnownIssues()
 void JASPVersionChecker::downloadVersionFinished()
 {
 	QString version			= _networkReply->readAll().trimmed(),
-			downloadfile	= "https://jasp-stats.org/download/";
+			downloadfile	= "http://jasp-stats.org/download/";  //Do not do use https here because on windows some PC has ssl verify problems.
```

Comment:
This is only used when one clicks the download button.
Id rather not remove https here, as that could allow mitm to lead people to the wrong (possibly spoofed) downloadpage

Reply:
Ah, yes!